scripts/experiment_notebooks/preprocess.ipynb

```diff
@@ -0,0 +1,294 @@
+{
+ "cells": [
```

Comment:
Are these notebooks different than the `preprocess` or `train` scripts in `scripts/`?

Reply:
not majorly, it just takes borders into account. We can edit the scripts to do that as well. I thought we want both notebooks and scripts from previous discussions, but we can keep one version only.

Reply:
It just seems confusing to have both, unless sufficient context is given. Can you make it clear that this is for the permutation experiments, either by putting it in an appropriate folder or creating a readme?

Reply:
It's not for the permutation specifically. It's just adds the target_locs saving (for fixed runs) and add the border bit. I'll remove it if it causes a confusion. 
the current script might need minor changes to reflect the code though.

Reply:
Hm, I don't think it's worth postponing the merge just because of this issue. We should create a future task of reconciling the scripts. I suspect that this notebook supersedes the other scripts, but it would still take some attention to check.